bin/cli.js

```diff
@@ -40,6 +42,13 @@ const browserLaunchers = {
   'Safari':    'karma-safari-launcher'
 };
 
+const logLevels = ['disable', 'default', 'debug'];
```

Comment:
I'd prefer for `'disable'` to be named `'quiet'` as that's much more common name for such case.